tempodb/encoding/vparquet3/schema.go

```diff
@@ -472,7 +483,7 @@ func traceToParquet(meta *backend.BlockMeta, id common.ID, tr *tempopb.Trace, ot
 
 				ss.Events = extendReuseSlice(len(s.Events), ss.Events)
 				for ie, e := range s.Events {
-					eventToParquet(e, &ss.Events[ie])
+					eventToParquet(e, &ss.Events[ie], s.StartTimeUnixNano)
```

Comment:
`nitpick` The function `eventToParquet()` receives the event start time in order to calculate `TimeSiceStartNano`. The reverse function ` parquetToProtoEvents()` receives the whole span. It would be more consistent to pass the span or only the start time to both (I personally favor only passing in the start time)